fuzzer/Cargo.toml

```diff
@@ -17,6 +17,8 @@ honggfuzz = "0.5.55"
 libfuzzer-sys = "0.4"
 cairo-vm = { path = "../vm", features = ["arbitrary"] }
 cairo-felt = { path = "../felt", features = ["arbitrary"] }
+serde = { version = "1.0.180", features = ["derive"] }
+serde_json = "1.0.104"
```

Comment:
can we use the `serde_json = { workspace = true } `version?
same for the serde import

Reply:
I have a commit ready to go with the updated `Cargo.toml`s to include the fuzzer directory into the list of workspaces. Let me know if I should push it

Reply:
Thanks @Juan-M-V !
I think that for now, it is better to keep it aside from the workspace
I would like to know what you think about that?